presto-docs/src/main/sphinx/release/release-0.259.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.259
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a casting error that sometimes occurred for EXPLAIN queries when the query plan contained LIKE predicates.
+* Add Weibull distribution CDF :func:`weibull_cdf` and inverse CDF :func:`inverse_weibull_cdf` functions.
+* Enable verbose error messages for ``EXCEEDED_LOCAL_MEMORY_LIMIT`` by default.  This can be disabled by setting the property ``memory.verbose-exceeded-memory-limit-errors-enabled`` to ``false``.
+* Add support for ``CREATE TYPE`` syntax.
+
+JDBC Changes
```

Comment:
In my opinion - there are 2 things - `JDBC` (corresponding to presto-jdbc module) and `JDBC Connector` (presto-base-jdbc module) and we should differentiate between them under 2 headings. The first entry is relevant to `JDBC` and the second is relevant to `JDBC Connector`.

Thoughts @rschlussel ?

Reply:
yup, good point.

Reply:
Separated then in to `JDBC Driver changes` and `JDBC changes`

Reply:
`JDBC Driver Changes` and `JDBC Connector Changes`